pkg/accounting/accounting.go

```diff
@@ -103,16 +135,21 @@ func NewAccounting(
 	Logger logging.Logger,
 	Store storage.StateStorer,
 	Pricing pricing.Interface,
+	refreshRate *big.Int,
 ) (*Accounting, error) {
 	return &Accounting{
 		accountingPeers:  make(map[string]*accountingPeer),
 		paymentThreshold: new(big.Int).Set(PaymentThreshold),
 		paymentTolerance: new(big.Int).Set(PaymentTolerance),
 		earlyPayment:     new(big.Int).Set(EarlyPayment),
+		disconnectLimit:  new(big.Int).Add(PaymentThreshold, PaymentTolerance),
 		logger:           Logger,
 		store:            Store,
 		pricing:          Pricing,
 		metrics:          newMetrics(),
+		refreshRate:      refreshRate,
+		timeNow:          time.Now,
+		minimumPayment:   new(big.Int).Div(refreshRate, big.NewInt(5)),
```

Comment:
Consider setting this as a constant, for readability

Reply:
Please also add the reasoning behind this specific value of the `minimumPayment`

Reply:
done